packages/constants-internal/src/constants.ts

```diff
@@ -283,3 +283,28 @@ export const STATUS = {
   SUCCESS: 'success',
   ERROR: 'error',
 };
+
+/**
+ * Constants representing the help text for
+ * each view.
+ */
+export const ViewHelpMapping = {
```

Comment:
In order for this to be possible to import like
```js
import { ViewHelpMapping } from '@vitessce/constants-internal';
```
(as you have used in each of the views),

the variable `ViewHelpMapping` also needs to be exported from the `index.js` file located at `packages/constants-internal/src/index.js`. Otherwise it will not be exposed as an exported variable that is available to import from the sub-package `@vitessce/constants-internal`.